src/Spout/Writer/Common/Creator/WriterEntityFactory.php

```diff
@@ -25,6 +38,50 @@ public static function createWriter($writerType)
         return (new WriterFactory())->create($writerType);
     }
 
+    /**
+     * @return \Box\Spout\Writer\CSV\Writer
+     */
+    public static function createCSVWriter()
```

Comment:
Why not just calling `return self::createWriter(Type:CSV)`? There's no need to redefine how a CSV writer should be constructed here. That's what I meant with my previous comments.
Also, you'll need to add/update tests.

Reply:
Actually the factory design is not good enough when you want to make the PHPDoc correct and let IDE suggest you the methods special for one certain type of writer. Anyway, it is of course able to be realized by manually call the create codes, is not better to provide them rather than a vague factory?